### PR TITLE
ML-039/add carrierwave

### PR DESCRIPTION
**Pull request by twill11**
# Description

Add carrierwave
Store pictures
Add upload from url
Add uploaders tests

## Trello card link

https://trello.com/c/gpML94eX/37-ml-039-add-carrierwave

# Checklist:

- [x] My code fixes the problem or adds required functionality
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [x] I have added tests that prove that my feature works
- [x] New and existing unit tests pass locally with my changes

# Additional requirements:

- [x] Does your code require any migrations to be ran?
- [x] Does your code require installing any new gems or system dependencies?
- [x] Does your code require any changes to current deployment process?

# Additional Information

### CODE REVIEW

File: config/initializers/carrierwave.rb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+CarrierWave.configure do |config|
+  config.cache_storage = :file
+end
```
Comment (dmitrymalllpr) [inline config/initializers/carrierwave.rb:3]: add line

**Review by dmitrymalllpr — CHANGES_REQUESTED** [review]

File: app/uploaders/poster_uploader.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
+class PosterUploader < CarrierWave::Uploader::Base
```
Comment (WhatWasThatt) [inline app/uploaders/poster_uploader.rb:1]: repeated code

**Review by WhatWasThatt — COMMENTED** [review]
can you update the tests with the new changes?

**Review by WhatWasThatt — APPROVED** [review]

File: app/controllers/api/v1/movies_controller.rb [hunk 3]
```diff
@@ -24,7 +24,7 @@ def create
       end
 
       def update
-        if @movie.update(movie_params)
+        if @movie.update!(movie_params)
```
Comment (dmitrymalllpr) [inline app/controllers/api/v1/movies_controller.rb:27]: Why did you use bang ?

**Review by dmitrymalllpr — CHANGES_REQUESTED** [review]

**Review by dmitrymalllpr — APPROVED** [review]

**Review by KarinaKlevzhits — APPROVED** [review]

**Pull request merged**